### ISSUE TITLE
Issue with recipe checking with JEI when a recipe has more than one input item

### ISSUE THREAD

**Issue by jamesjimmy88** [Frinn38/Custom-Machinery#35]
When a custom machine has multiple inputs, the use check for JEI only works for the last input item.
For example, with this script:
```
mods.custommachinery.CMRecipeBuilder.create("custommachinery:testingmachine", 20)
.requireItem(<item:minecraft:dirt>*4)
.requireItem(<item:minecraft:stone>*8)
.requireItem(<item:minecraft:stick>*4)
.requireEnergy(2000)
.produceItem(<item:minecraft:diamond>)
.priority(80)
.build();
```
It's possible to view the recipe only by checking the recipes for diamonds or by checking the uses for sticks or for the custom machine. Checking the uses for dirt or stone won't work (it will still show the other uses for the items).
By checking the recipes and uses I mean using R or U (or left/right click) on the items in JEI

Tested with CM 1.16.5-0.5.5, JEI 1.16.5-7.7.1.138 on Forge 36.2.22

**Comment by Frinn38** [comment]
Yeah someone reported me this issue on discord a while ago, this is already fixed in dev and will be in the next update.
Leaving this issue open until the update, in case someone else encounter this bug.

**Comment by Frinn38** [comment]
Fixed in 0.5.6
